bool request_for_GPS
bool request_to_update_reference_GPS
sensor_msgs/NavSatFix reference_GPS
---
sensor_msgs/NavSatFix tile_origin_GPS 
geometry_msgs/Point tile_origin_xyz_wrt_reference_GPS
sensor_msgs/NavSatFix[] GPS_output
geometry_msgs/Point[] Transformed_points
